spinalcordtoolbox/scripts/sct_analyze_lesion.py

```diff
@@ -393,14 +356,62 @@ def _measure_axial_damage_ratio(self, im_data, p_lst, idx):
         printv('  Maximum axial damage ratio: ' + str(np.round(maximum_axial_damage_ratio, 2)),
                self.verbose, type='info')
 
+    def _measure_interpolated_tissue_bridges(self, tissue_bridges_df, p_lst, idx):
+        """
+        Compute the interpolated tissue bridges. These bridges are computed from two sagittal slices, which are
+        interpolated to the midsagittal slice.
+        """
+        # Interpolated tissue bridges
+        interpolated_dorsal_bridge_width_mm = {}
+        interpolated_ventral_bridge_width_mm = {}
+        # Loop across unique axial slices (S-I direction) to interpolate the tissue bridges from the two sagittal
+        # slices. Note that here, we get axial slices with the lesion only for the sagittal slices used for the
+        # interpolation, i.e., not for all sagittal slices containing the lesion.
+        axial_slices = np.unique(tissue_bridges_df[tissue_bridges_df['sagittal_slice'].isin(self.interpolation_slices)]
+                                 ['axial_slice'])
+
+        # If there are no axial slices, it means that the current lesion is not captured on the sagittal slices used
+        # for the interpolation. In this case, we will not compute the interpolated tissue bridges.
+        if len(axial_slices) == 0:
+            self.measure_pd.loc[idx, 'interpolated_dorsal_bridge_width [mm]'] = np.nan
+            self.measure_pd.loc[idx, 'interpolated_ventral_bridge_width [mm]'] = np.nan
+            return
+
+        for axial_slice in axial_slices:
+            # Get bridges for the 3 sagittal slices used for the interpolation
```

Comment:
Nit: 3 -> 2?

Reply:
good catch 👍🏻 fixed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4702/commits/4727c354f03f1b63b807132d09f68bc45c009ca8